app/services/gogovan.rb

```diff
@@ -13,6 +13,7 @@ def initialize(user = nil, options = {})
     @need_carry   = options['needCarry']
     @district_id  = options['districtId']
     @vehicle      = options['vehicle']
+    @ggv_uuid     = options['ggv_uuid']
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.